modules/sugar/src/test/ts/browser/LocationTest.ts

```diff
@@ -136,7 +136,8 @@ UnitTest.asynctest('LocationTest', (success, failure) => {
     pos = SugarLocation.viewport(body);
     Assert.eq('', 0, pos.top);
     Assert.eq('', 0, pos.left);
-    Assert.eq('scroll bar width, got=' + scrollBarWidth, true, scrollBarWidth > 5 && scrollBarWidth < 50 || (platform.os.isMacOS() && scrollBarWidth === 0));
+    const noVisibleScrollbarBrowser = platform.os.isMacOS() || (platform.browser.isFirefox() && platform.os.isLinux()) && (platform.os.isWindows() && platform.os.version.major >= 11);
```

Comment:
```suggestion
    const noVisibleScrollbarBrowser = platform.os.isMacOS() || (platform.browser.isFirefox() && platform.os.isLinux()) || (platform.browser.isFirefox() && platform.os.isWindows() && platform.os.version.major >= 11);
```

Reply:
I fixed another bug here whereby the check used an `&&` instead of `||` for the different linux/windows checks.

Reply:
Has this condition been determined empirically, or is it documented somewhere in a Firefox changelog?

Reply:
It's documented. However we should probably feature detect this instead since it's depends on settings in Firefox or the OS. But that is a later problem.